Libplanet/Serialization/BencodexFormatter.cs

```diff
@@ -192,6 +193,8 @@ private IValue ToBencodexValue(object o)
                     return new Integer(bi);
                 case byte[] byteArray:
                     return ToBencodexKey(byteArray);
+                case ImmutableArray<byte> immutableByteArray:
```

Comment:
I think it would be better if this conversion logic goes to `RawTransaction.GetObjectData()` because `byte[]` → `ImmutableArray<byte>` was covered in `RawTranscation(SerializationInfo info, StreamingContext context)`.

Reply:
Serializing at `ISerializable.GetObjectData(SerializationInfo info, StreamingContext context)` and Deserializing at `RawTranscation(SerializationInfo info, StreamingContext context)`.
Now I get it.